doc/user-guide/io.rst

```diff
@@ -1151,6 +1202,7 @@ The simplest way to serialize an xarray object is to use Python's built-in pickl
 module:
 
 .. ipython:: python
+    :okexcept:
```

Comment:
Why this?

Reply:
It generates this error, 
```
updating environment: [config changed ('autodoc_type_aliases')] 1370 added, 0 changed, 0 removed
WARNING: ources... [ 99%] user-guide/io                                                                                                                                           
>>>-------------------------------------------------------------------------
Exception in /home/anurag/Code/GSoC/xarray/doc/user-guide/io.rst at block ending on line 1213
Specify :okexcept: as an option in the ipython:: block to suppress this message
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[65], line 1
----> 1 pkl = pickle.dumps(ds, protocol=-1)

TypeError: cannot pickle '_json.Scanner' object

<<<-------------------------------------------------------------------------

Exception occurred:
  File "/home/anurag/anaconda3/envs/xarray-docs/lib/python3.10/site-packages/IPython/sphinxext/ipython_directive.py", line 584, in process_input
    raise RuntimeError(
RuntimeError: Unexpected exception in `/home/anurag/Code/GSoC/xarray/doc/user-guide/io.rst` line 1213
The full traceback has been saved in /tmp/sphinx-err-cjtp2udz.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker
```

Reply:
Perhaps ping whoever added that piece.